hieradata/common.yaml

```diff
@@ -599,6 +599,7 @@ govuk_ci::master::pipeline_jobs:
   govuk_navigation_helpers: {}
   govuk-provisioning:
     source: 'git'
+  govuk-puppet: {}
```

Comment:
This was added manually because this list is also used by the "Deploy App" Jenkins job, and we don't want govuk-puppet to be in that job. It will just fail, but we need to be wary of where this list is used elsewhere.